### DISCUSSION TITLE
Possible to Ignore headers in Table of Contents?

### DISCUSSION THREAD

**Discussion by Scorch-Dev** [requarks/wiki#3803] [Help / Questions]
Is there currently a way to ignore headers in the markdown editor so they don’t appear in the table of contents? E.g. adding something like

```
# ignored header {.toc-ignore}
```

I tried the above, but it didn’t work for me. Also tried {. unnumbered .unlisted} just for the sake of it, but no such luck.

**Comment by tukusejssirs** [comment]
I am looking for the same, but for HTML files. Actually, I need to ignore headings with `tabset` classes (tabs), which output blank items in the TOC.

That said I still think that it might that the ability of ingoring some heading in the TOC _and_ limiting the heading level (like up to `<h2>` including) is a very useful and needed feature.
